soepy/test/test_unit.py

```diff
@@ -1,11 +1,14 @@
 import numpy as np
+from collections import namedtuple
```

Comment:
This is from the standard library and numpy is not, please keep the ordering according to PEP8.